Casks/v/veusz.rb

```diff
@@ -12,10 +15,13 @@
     url :url
     regex(/^veusz[._-]v?(\d+(?:\.\d+)+)(?:[._-].+)?\.dmg$/i)
     strategy :github_latest do |json, regex|
+      tag_version = json["tag_name"]&.gsub("veusz-", "")
       json["assets"]&.map do |asset|
         match = asset["name"]&.match(regex)
         next if match.blank?
 
+        next "#{match[1]},#{tag_version}" if match[1] != tag_version
+
         match[1]
```

Comment:
```suggestion
      tag_version = json["tag_name"]&.[](/^veusz[._-]v?(\d+(?:\.\d+)+(?:-fix)?)$/i, 1)
      next if tag_version.blank?

      json["assets"]&.map do |asset|
        match = asset["name"]&.match(regex)
        next if match.blank?

        (match[1] == tag_version) ? tag_version : "#{match[1]},#{tag_version}"
```

This updates the block to align with our usual pattern (e.g., [`windterm`](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/w/windterm.rb)).